configs/stable_diffusion/stable-diffusion_ddim_denoisingunet.py

```diff
@@ -80,4 +80,6 @@
         subfolder='text_encoder'),
     tokenizer=stable_diffusion_v15_url,
     scheduler=diffusion_scheduler,
-    test_scheduler=diffusion_scheduler)
+    test_scheduler=diffusion_scheduler,
+    tomesd_cfg=dict(ratio=0.5),
```

Comment:
I suggest keeping this config unchanged and adding a new config called `stable-diffusion_ddim_denoising-tomesd_0.5.py`.

Reply:
OK!